featureFlag/style.css

```diff
@@ -1,3 +1,5 @@
+@import url('../global.css');
```

Comment:
Import statement might not be ideal here, can this be changed?
refer [this](https://www.debugbear.com/blog/avoid-css-import#:~:text=The%20CSS%20%40import%20rule%20can,to%20take%20longer%20to%20render.)

Reply:
> Import statement might not be ideal here, can this be changed? refer [this](https://www.debugbear.com/blog/avoid-css-import#:~:text=The%20CSS%20%40import%20rule%20can,to%20take%20longer%20to%20render.)

Great observation and well-suggested guideline @TanishqSingla  🙌🏻